api/src/main/java/org/openmrs/module/labintegration/api/hl7/scc/SCCHL7Config.java

```diff
@@ -0,0 +1,37 @@
+package org.openmrs.module.labintegration.api.hl7.scc;
+
+import org.openmrs.Order;
+import org.openmrs.module.labintegration.api.hl7.config.AbstractHL7Config;
+import org.openmrs.module.labintegration.api.hl7.config.OrderIdentifier;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SCCHL7Config extends AbstractHL7Config {
+	
+	private static final String SENDING_FACILITY_NAMESPACE_ID = "LNSP";
+	
+	@Override
+	public String getReceivingApplication() {
+		return null;
```

Comment:
Do we have some default values?

Reply:
It's just not-needed field in the message, like above

Reply:
Okay.